elisp-refs.el

```diff
@@ -427,11 +427,11 @@ at least one line has no indent.
 
 STRING should have a 'elisp-refs-start-pos property. The returned
 string will have this property updated to reflect the unindent."
-  (let* ((lines (s-lines string))
-         ;; Get the leading whitespace for each line.
-         (indents (--map (car (s-match (rx bos (+ whitespace)) it))
-                         lines))
-         (min-indent (-min (--map (length it) indents))))
+  (let* ((lines (split-string string "\n"))
+         (min-indent most-positive-fixnum) (indent 0))
```

Comment:
- Elisp has bignums now so I would find it more elegant to do away with the arbitrary fixnum limit, e.g. by continuing to use Dash helpers (min-by, reduce), or by using `nil` or similar as an initial value.

- The `indent` variable can be moved into the loop to avoid the need for `setq` (which should make it slightly more efficient too).